app/src/index.tsx

```diff
@@ -3,35 +3,53 @@ import * as ReactDOM from 'react-dom';
 import App from './components/App';
 import * as serviceWorker from './serviceWorker';
 
-import { Drizzle, DrizzleOptions, generateStore } from 'drizzle';
-import { DrizzleContext } from 'drizzle-react';
-
 import Governance from './contracts/Governance.json';
 import ArtifactApplication from './contracts/ArtifactApplication.json';
 import ArtifactRegistry from './contracts/ArtifactRegistry.json';
 import Artists from './contracts/Artists.json';
 
 import './theme.scss';
+import Web3 from 'web3';
 
-const options: DrizzleOptions = {
-  contracts: [ArtifactApplication, Governance, ArtifactRegistry, Artists],
-  web3: {
-    fallback: {
-      type: 'ws',
-      url: 'ws://127.0.0.1:8545',
-    },
-  },
-};
+// eslint-disable-next-line
+const contract = require('@truffle/contract');
+
+const doDapp = async (): Promise<void> => {
+  console.log('beabadoobeeeee');
 
-const drizzleStore = generateStore({ drizzleOptions: options });
-const drizzle = new Drizzle(options, drizzleStore);
+  const web3 = new Web3('ws://127.0.0.1:8545');
+  // this is recommended way of doing it
+  // but it stopped working when i removed drizzle
+  // const web3 = new Web3(Web3.givenProvider || 'ws://127.0.0.1:8545');
```

Comment:
Can you debug this further? It should definitely work?

Reply:
i can but not today